gps/app/locales/ml.json

```diff
@@ -19,15 +24,21 @@
   },
   "import": {
     "button_text": "പഴയ ലൊക്കേഷനുകൾ ഇംപോർട്ട് ചെയ്യുക",
+    "error": "നിങ്ങളുടെ ഡാറ്റ ഇമ്പോർട്ടുചെയ്യുമ്പോൾ എന്തോ തെറ്റായി സംഭവിച്ചു.",
     "google": {
+      "already_imported": "നൽകിക്കഴിഞ്ഞ് ടേക്ക് ഔട്ട് ഫയൽ ഇതിനകം തന്നെ ഇറക്കുമതി ചെയ്തിട്ടുണ്ട്.",
       "disclaimer": "COVID സെയ്ഫ് പാത്ത്-ന് Google മായി ഒരു ബന്ധവുമില്ല കൂടാതെ നിങ്ങളുടെ ഡാറ്റ ഒരിക്കലും പങ്കിടില്ല.",
-      "instructions_detailed": "ഗൂഗിൾ ടേക്ഔട്ട് സന്ദർശിക്കുക, ഇനിപ്പറയുന്ന ക്രമീകരണങ്ങൾ ഉപയോഗിച്ചുകൊണ്ട് നിങ്ങളുടെ ലൊക്കേഷൻ ചരിത്രം എക്സ്പോർട്ട് ചെയ്യുക: \n1. ഡെലിവറി രീതി: \"ഡ്രൈവ് ചെയ്യാൻ ചേർക്കുക\" \n2. ആവൃത്തി: \"ഒരിക്കൽ എക്സ്പോർട്ട് ചെയ്യുക\" \n3. ഫയൽ തരം & വലിപ്പം: \". തപാൽ\", \"1GB\"\n4. കയറ്റുമതി തയ്യാറാകുമ്പോൾ ഗൂഗിൾ ഒരു ഇമെയിൽ അയയ്ക്കുന്നു \n5. ലൊക്കേഷനുകള് ഇറക്കുമതി ചെയ്യാന് ഇവിടെ തിരിച്ചെത്തും. ഇംപോര് ട്ട് ഉപാധികള്:\n-ഗൂഗിൾ ഡ്രൈവിൽ നിന്ന് ഇംപോർട്ട് ചെയ്യുക\n-ബ്രൗസറിൽ നിന്ന് ഡൗൺലോഡ് ചെയ്യുക, തുടർന്ന് പ്രാദേശിക ഫോൺ ഫയലുകളിൽ നിന്ന് ഇറക്കുമതി ചെയ്യുക. ഫയലുകൾ വലുതാക്കാം എന്നതിനാൽ വൈഫൈ നെറ്റ് വർക്കിലേക്ക് ഉറപ്പാക്കുക.",
+      "file_open_error": "ഫയൽ തുറക്കാൻ കഴിഞ്ഞില്ല. \n ദയവായി, Google ഡ്രൈവിൽ നിന്ന് ഫയൽ തുറന്നിട്ടുണ്ടെന്ന് ഉറപ്പാക്കുക",
+      "instructions_detailed": "ഗൂഗിൾ ടേക്ഔട്ട് സന്ദർശിക്കുക, ഇനിപ്പറയുന്ന ക്രമീകരണങ്ങൾ ഉപയോഗിച്ചുകൊണ്ട് നിങ്ങളുടെ ലൊക്കേഷൻ ചരിത്രം എക്സ്പോർട്ട് ചെയ്യുക: \n1. ഡെലിവറി രീതി: \"ഡ്രൈവ് ചെയ്യാൻ ചേർക്കുക\" \n2. ആവൃത്തി: \"ഒരിക്കൽ എക്സ്പോർട്ട് ചെയ്യുക\" \n3. ഫയൽ തരം & വലിപ്പം: \".zip\",\"1 GB\"\n4. കയറ്റുമതി തയ്യാറാകുമ്പോൾ ഗൂഗിൾ ഒരു ഇമെയിൽ അയയ്ക്കുന്നു \n5. ലൊക്കേഷനുകള് ഇറക്കുമതി ചെയ്യാന് ഇവിടെ തിരിച്ചെത്തും. ഇംപോര്ട്ട് ഉപാധികള്:\n-ഗൂഗിൾ ഡ്രൈവിൽ നിന്ന് ഇംപോർട്ട് ചെയ്യുക\n-ബ്രൗസറിൽ നിന്ന് ഡൗൺലോഡ് ചെയ്യുക, തുടർന്ന് പ്രാദേശിക ഫോൺ ഫയലുകളിൽ നിന്ന് ഇറക്കുമതി ചെയ്യുക. ഫയലുകൾ വലുതാക്കാം എന്നതിനാൽ വൈഫൈ നെറ്റ് വർക്കിലേക്ക് ഉറപ്പാക്കുക.",
       "instructions_first": "Google- ൽ നിന്ന് ലൊക്കേഷൻ ഡാറ്റ ചേർക്കുന്നത് നിങ്ങളുടെ സമീപകാല ലൊക്കേഷനുകൾ നിർമ്മിക്കുന്നതിനുള്ള ഒരു തുടക്കം നൽകും.",
       "instructions_second": "നിങ്ങൾക്ക് ഇംപോർട്ട് ചെയ്യാൻ കഴിയും മുമ്പ്, ഗൂഗിളിൽ നിന്നുള്ള നിങ്ങളുടെ ലൊക്കേഷൻ ഡാറ്റ ആദ്യം \"പുറത്തെടുക്കണം\".",
+      "invalid_file_format": "നൽകിയ ഫയൽ ഫോർമാറ്റ് പിന്തുണയ്ക്കുന്നില്ല. \n പിന്തുണയ്ക്കുന്ന ഫോർമാറ്റുകൾ: \".zip\".",
```

Comment:
same comment about the space after line break char

Reply:
Fix these in lokalise. I don't want to do manual edits here as well. It's extra work for little gain,. And I want to be able to automate this process soon.